pkgs/servers/varnish/default.nix

```diff
@@ -13,8 +13,8 @@ let
 
       nativeBuildInputs = [ pkgconfig ];
       buildInputs = [
-        pcre libxslt groff ncurses readline python3 libedit
-        python3.pkgs.docutils makeWrapper
+        pcre libxslt groff ncurses readline python libedit
+        python.pkgs.docutils makeWrapper
       ] ++ extraBuildInputs;
```

Comment:
`python`, `python.pkgs.docutils`, `makeWrapper` seem to be in `nativeBuildInputs` ?

Reply:
@jonringer do you mind addressing this? I will merge it afterwards.

Reply:
Yea, i was also looking into some of the plugins, they seem to be in an odd state